src/services/bookmarkService.js

```diff
@@ -0,0 +1,60 @@
+const { v4: uuidv4 } = require("uuid");
+const { db } = require("../db/db");
+const { bookmark } = require("../db/schema");
+const { eq } = require("drizzle-orm");
+const { bookmarkSchema } = require("../utils/joiSchema");
+const Joi = require("joi");
+
+const getAllBookmarks = async () => {
+  try {
+    const result = await db.select().from(bookmark);
+    return result;
+  } catch (error) {
```

Comment:
While errors are caught, they're simply rethrown. Consider adding more specific error messages or logging.

```
catch (error) { 
    if (error instanceof Joi.ValidationError) { 
      throw new Error(`Invalid bookmark ID: ${error.message}`);
    }
    throw new Error(`Database error: ${error.message}`);
}
```

Reply:
this one also I will cover in the error handling PR to not make this PR huge